client/src/components/ProfileCover/ProfileCover.jsx

```diff
@@ -7,64 +7,70 @@ import PropTypes from 'prop-types'
 import EditOutlinedIcon from '@material-ui/icons/EditOutlined'
 import UpdateProfile from '../UpdateProfile/UpdateProfile'
 
-const ProfileCover = ({ user }) => {
+const ProfileCover = ({ user, profileTab, handleChangeTab }) => {
   const { avatar, firstName, lastName, profileCover } = user
-  const classes = useStyles({profileCover: profileCover.src})
+  const classes = useStyles({ profileCover: profileCover.src })
 
-  const [value, setValue] = useState('profile')
   const [ modalOpen, setModalOpen ] = useState(false)
 
-  const handleChange = (event, newValue) => {
-    setValue(newValue)
-  }
-
   const handleModal = () => {
     setModalOpen(!modalOpen)
   }
 
   return (
     <Fragment>
-      <div className={classes.avatarBg}>
-        <Avatar className={classes.avatarImg} src={avatar.src} />
-        <p className={classes.avatarName}>{firstName} {lastName}</p>
-        <Button
-          variant='contained'
-          onClick={handleModal}
-          className={classes.editProfileBtn}
+      <div className={classes.container}>
```

Comment:
It's hard to see from changes list, but if this div wraps whole component, we don't need `<Fragment>`